crates/core/src/contract/storages/sqlite.rs

```diff
@@ -28,7 +28,7 @@ impl Pool {
     #[cfg_attr(feature = "redb", allow(unused))]
     pub async fn new(db_dir: Option<&Path>) -> Result<Self, SqlDbError> {
         let opts = if let Some(db_dir) = db_dir {
-            let file = db_dir.join("freenet.db");
```

Comment:
did you try running with sqlite to see things initialise properly?

Reply:
I tried to use sqlx with sqlite only in standalone project, and find that this only a file. But not try it with the freenet cli.

Reply:
Compile and install locally freenet to use SQLite and not Redb, install and try run ping contract to see everything works correctly before merging

Reply:
@iduartgomez @al8n Still relevant?

Reply:
Yes please leave this open